ddtrace/internal/telemetry/writer.py

```diff
@@ -431,45 +432,47 @@ def _app_started_event(self):
         if self._forked:
             # app-started events should only be sent by the main process
             return
+        #  List of configurations to be collected
```

Comment:
Instead of this comment, you could name the list variable `configurations_to_collect`. This would make the code more self-describing, with the information from this comment following the variable throughout its lifespan.